generators/app/templates/README.md

```diff
@@ -10,88 +14,100 @@ This README is intended to help get you started. Definitely update and improve t
-    convert me <expression> to <units> - Convert expression to given units.
-    help - Displays all of the help commands that Hubot knows about.
+    <%= botName %> animate me <query> - The same thing as `image me`, except adds [snip]
+    <%= botName %> help - Displays all of the help commands that <%= botName %> knows about.
     ...
 
 
 ### Scripting
 
 An example script is included at `scripts/example.coffee`, so check it out to
-get started, along with the [Scripting Guide](https://github.com/github/hubot/blob/master/docs/scripting.md).
+get started, along with the [Scripting Guide](scripting-docs).
 
 For many common tasks, there's a good chance someone has already one to do just
 the thing.
 
-### hubot-scripts
+[scripting-docs]: https://github.com/github/hubot/blob/master/docs/scripting.md
 
-There will inevitably be functionality that everyone will want. Instead
-of writing it yourself, you can check
-[hubot-scripts][hubot-scripts] for existing scripts.
+### external-scripts
 
-To enable scripts from the hubot-scripts package, add the script name with
-extension as a double quoted string to the `hubot-scripts.json` file in this
-repo.
+There will inevitably be functionality that everyone will want. Instead of
+writing it yourself, you can use existing plugins.
 
-[hubot-scripts]: https://github.com/github/hubot-scripts
+Hubot is able to load plugins from third-party `npm` packages. This is the
+recommended way to add functionality to your hubot. You can get a list of
+available hubot plugins on [npmjs.com](npmjs) or by using `npm search`:
+
+    % npm search hubot-scripts panda
+    NAME             DESCRIPTION                        AUTHOR DATE       VERSION KEYWORDS
+    hubot-pandapanda a hubot script for panda responses =missu 2014-11-30 0.9.2   hubot hubot-scripts panda
+    ...
 
-### external-scripts
 
-Hubot is able to load scripts from third-party `npm` package. Check the package's documentation, but in general it is:
+To use a package, check the package's documentation, but in general it is:
 
-1. Add the packages as dependencies into your `package.json`
-2. `npm install` to make sure those packages are installed
-3. Add the package name to `external-scripts.json` as a double quoted string
+1. Use `npm install --save` to add the package to `package.json` and install it
+2. Add the package name to `external-scripts.json` as a double quoted string
 
 You can review `external-scripts.json` to see what is included by default.
 
+[npmjs]: https://www.npmjs.com
+
+### hubot-scripts
+
+Before hubot plugin packages were adopted, most plugins were held in the
+[hubot-scripts][hubot-scripts] package. Some of these plugins have yet to be
+migrated to their own packages. They can still be used but the setup is a bit
+different.
+
+To enable scripts from the hubot-scripts package, add the script name with
+extension as a double quoted string to the `hubot-scripts.json` file in this
```

Comment:
I will have to verify, but I think there is a way to use the hubot-scripts package through external-scripts.json, where you can specify which scripts to load. Instead of an array, it can be an hash with the key being the package name, and the value either being a string like '*' to mean all scripts, or an array of scripts.

That might help reduce the number of options and config files people have to consider